### ISSUE TITLE
Modal on bottom & max width doesnt work

### ISSUE THREAD

**Issue by peroxovy** [wire-elements/modal#348]
Hello, I've found the issue, that whenever I've use:

```php
    public static function modalMaxWidth(): string
    {
        return '2xl';
    }
```

Inside ModalComponent, the changes did not apply and the Modal is a `w-full` component on the page. I can't change the setting anyhow.

The only workaround is to edit `vendor/wire-elements/modal/resouces/views/modal.blade.php`, and change delete from the line 43 the class `w-full` from `<div id="modal-container">`:

![image](https://github.com/wire-elements/modal/assets/62743726/ce4d2339-7237-4d98-a907-0d462cf86dac)

Another issue is that, I had the modals all the time on the bottom of the page, even if it's desktop/mobile. The workaround is to change on the line 17 `justify-end` to `justify-center`:

![image](https://github.com/wire-elements/modal/assets/62743726/196f2092-a935-4d96-810f-71f6211f72bb)

But the question is, why is this happening? Modifying files in vendor is a bad behaviour, so I'm asking if there's possibility to add a feature of publishing views to have more control around the modals?

If you have any other suggestion or thoughts, please comment.

**Comment by PhiloNL** [comment]
Did you add the blade files to the `tailwind.config.js`? You can already publish the views `php artisan vendor:publish`.

**Comment by peroxovy** [comment]
> Did you add the blade files to the `tailwind.config.js`? You can already publish the views `php artisan vendor:publish`.

Yes, all files including those from your package are included to `tailwind.config.js`.

If I use default screen settings for TailwindCSS (commenting out my custom, and getting back to `w-full` inside `vendor/.../modal.blade.php`), the `modalMaxWidth()` method is working.

Regarding the position of modal, issue is still present.

**Comment by PhiloNL** [comment]
The modal at the bottom is intended for mobile to make it easier to access but it shouldn't happen on desktop. I can't replicate this so I assume this might be related to your project.
